### PR TITLE
Update citation files and extend citation files for search engines

### PR DESCRIPTION
**Pull request by EmilyWes**
Exported 10 open access papers from various sources for various formats

### CODE REVIEW

**Comment by J535D165** [comment]
@EmilyWes thanks for this contribution. Can you update the README with the new search engines?

**Comment by EmilyWes** [comment]
@J535D165 Done!

File: Datasets/CSV/_baseline_abstrackr.csv [hunk 1]
```diff
@@ -0,0 +1,53 @@
+(internal) id,(source) id,pubmed id,keywords,abstract,title,journal,authors,tags,consensus,labeled_at,Emily,labeled_at,general notes (Emily),population notes (Emily),intervention/comparator notes (Emily),outcome notes (Emily)
```
Comment (J535D165) [inline Datasets/CSV/_baseline_abstrackr.csv]:
```suggestion
(internal) id,(source) id,pubmed id,keywords,abstract,title,journal,authors,tags,consensus,labeled_at,Anonymous,labeled_at,general notes (Anonymous),population notes (Anonymous),intervention/comparator notes (Anonymous),outcome notes (Anonymous)
```

Remove name.

**Comment by J535D165** [comment]
Would you like to update the submodule on https://github.com/asreview/asreview and open a PR? I'm curious if ASReview can load these files.

**Pull request merged**